extensions/arc/deployment/src/test/java/io/quarkus/arc/test/stereotype/DoubleScopedStereotypeTestCase.java

```diff
@@ -32,6 +35,7 @@ public void runTest() {
     @SessionScoped
     @RequestScoped
     @Stereotype
+    @Retention(RetentionPolicy.RUNTIME)
```

Comment:
This apparently breaks the `DoubleScopedStereotypeTestCase` :thinking:

Reply:
So, the `BeanRegistrarTest` actually has a [qualifier](https://github.com/quarkusio/quarkus/blob/main/independent-projects/arc/tests/src/test/java/io/quarkus/arc/test/buildextension/beans/BeanRegistrarTest.java#L278) violating this new check :)